rust/pact_ffi/src/verifier/handle.rs

```diff
@@ -65,6 +65,25 @@ impl VerifierHandle {
     self.provider.clone()
   }
 
+  /// Add a new transport to the verification process
+  pub fn add_transport(
+    &mut self,
+    protocol: String,
+    port: u16,
+    path: String,
+    scheme: Option<String>
+  ) {
+
+    let transport = ProviderTransport {
+      transport: protocol,
+      port: Some(port),
+      path: if path.is_empty() { None } else { Some(path) },
+      scheme: scheme
+    };
+
+    self.provider.transports.append(&mut vec![transport]);
```

Comment:
There must be a function to add a new item, as this one creates a new vector, converts it into a mutable slice, and then appends that to the vector

Reply:
Yeah, `push` is what we want to use

Reply:
I _swear_ this was the first thing I tried, but it seems to compile now so 🤷 . Fixed, thanks.